python/ray/import_thread.py

```diff
@@ -140,6 +140,16 @@ def _process_key(self, key):
                         "more discussion.", import_type, name,
                         ray_constants.DUPLICATE_REMOTE_FUNCTION_THRESHOLD)
 
+            if key.startswith(b"RemoteFunction"):
+                with profiling.profile("register_remote_function"):
+                    (self.worker.function_actor_manager.
+                     fetch_and_register_remote_function(key))
+            elif key.startswith(b"ActorClass"):
+                # Keep track of the fact that this actor class has been
+                # exported so that we know it is safe to turn this worker
+                # into an actor of that class.
+                self.worker.function_actor_manager.imported_actor_classes.add(
+                    key)
```

Comment:
This is copy-pasted. Maybe adjust these if statements and reduce duplicated code?
Also, the comment below should be updated.

Reply:
I refactored this function.